tf-modules/cumulus/variables.tf

```diff
@@ -526,3 +526,8 @@ variable "ems_deploy" {
   type        = bool
   default     = false
 }
+
+variable "region" {
+  type    = string
+  default = "us-east-1"
```

Comment:
see comment above re: the variable itself, but if this winds up being needed, having a default here should be noted as a breaking change as user configuration defaults would be directly impacted.    We should also consider us-west-(1,2?) as that where most of our customers are deploying.

Reply:
This should be removed after 01fa54f2a5ef5942c98dd60ca27051c9a39c2bf0

Reply:
Whoops! Not sure how that got back in. Let me update.